spec/features/admin_can_edit_an_item_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+RSpec.describe 'admin' do
+  context 'with valid credentials' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.